### PR TITLE
For checking

### PR DESCRIPTION
**Pull request by EuhenySiniy**

### CODE REVIEW

File: src/Lists/MyArrayList.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
+package Lists;
+
+import java.util.Arrays;
+
+public class MyArrayList<E> implements List<E> {
+    private int capacity = 10;
```
Comment (Taniann) [inline src/Lists/MyArrayList.java]: better make it constant. and give name DEFAULT_CAPACITY

File: src/Lists/MyArrayList.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    private Object[] array;
+    private int indexLastElemet = -1;
```
Comment (Taniann) [inline src/Lists/MyArrayList.java]: why do you need index of last element if you have size?

File: src/Lists/MyArrayList.java [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    private int size;
+
+    public MyArrayList() {
+        array = new Object[capacity];
+    }
+
+    public MyArrayList(int size) {
```
Comment (Taniann) [inline src/Lists/MyArrayList.java]: what if size < 0 
better make check if it < default_capacity create array with default_capacity

File: src/Lists/MyLinkedList.java [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+        Node(Node<E> prev, E element, Node<E> next) {
+            this.item = element;
+            this.next = next;
+            this.prev = prev;
+        }
+    }
+
+    private int size;
+    private Node<E> prev;
```
Comment (Taniann) [inline src/Lists/MyLinkedList.java]: first

File: src/Lists/MyLinkedList.java [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+    private Node<E> next;
```
Comment (Taniann) [inline src/Lists/MyLinkedList.java]: last

**Comment by EuhenySiniy** [comment]
Thank you :)

**Pull request merged**